lib/foreman_maintain/concerns/system_helpers.rb

```diff
@@ -213,6 +213,13 @@ def proxy_plugin_name(plugin)
         ruby_prefix(scl) + proxy_plugin_prefix + plugin
       end
 
+      def hammer_plugin_name(plugin)
+        if debian?
+          plugin.tr!('_', '-')
+        end
```

Comment:
I'm not a fan of in place replacements. How about:
```suggestion
        plugin = plugin.tr('_', '-') if debian?
```

Reply:
I've changed it everywhere.